docs/misc/changelog.rst

```diff
@@ -64,6 +63,7 @@ Bug Fixes:
 - Fixed PPO train/n_updates metric not accounting for early stopping (@adamfrly)
 - Fixed loading of normalized image-based environments
 - Fixed ``DictRolloutBuffer.add`` with multidimensional action space (@younik)
+- Fixed ``VecExtractDictObs`` does not handle terminal observation (@WeberSamuel)
```

Comment:
I wasn't sure if this is the right place. Because this version is already released

Reply:
no sorry, if you re-open the PR with another branch I could fix that.

Reply:
(master is protected by default)

Reply:
okay, I will reopen the PR